src/Common/ThreadFuzzer.cpp

```diff
@@ -102,6 +103,10 @@ void ThreadFuzzer::initConfiguration()
     initFromEnv(sleep_probability, "THREAD_FUZZER_SLEEP_PROBABILITY");
     initFromEnv(sleep_time_us, "THREAD_FUZZER_SLEEP_TIME_US");
 
+    LOG_DEBUG(&Poco::Logger::get("ThreadFuzzer"), "Sleep probability {}", sleep_probability);
```

Comment:
Of course I'm going to remove this, just checking that env variables will be forwarded correctly.